app/src/main/java/chat/rocket/android/helper/ChatRoomRoleHelper.kt

```diff
@@ -0,0 +1,53 @@
+package chat.rocket.android.helper
+
+import chat.rocket.android.core.lifecycle.CancelStrategy
+import chat.rocket.android.db.DatabaseManager
+import chat.rocket.android.server.domain.CurrentServerRepository
+import chat.rocket.android.server.infrastructure.ConnectionManagerFactory
+import chat.rocket.common.model.RoomType
+import chat.rocket.common.model.roomTypeOf
+import chat.rocket.core.internal.rest.chatRoomRoles
+import chat.rocket.core.model.ChatRoomRole
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import timber.log.Timber
+import javax.inject.Inject
+
+class ChatRoomRoleHelper @Inject constructor(
```

Comment:
It is not acting as a helper class. It is mostly a presenter...

Reply:
@filipedelimabrito It helps in getting ChatRoles from chatroom id. It is used in PermissionsInterector. Should I move methods in PermissionsInterector? It will make PermissionsInterector long.